tests/test_syntax_errors.py

```diff
@@ -92,3 +92,7 @@ def test_sets(self):
                    '    a:\n'
                    '      set\n'
                    '...\n', None)
+
+    def test_non_printable_characters(self):
+        self.check('output: \x1b\n',
+                   None, problem=(1, 9))
```

Comment:
Please add more tests. At the least I can think of:
- invalid character in the first byte,
- invalid character in the last byte,
- invalid character not on line 1,
- multiple invalid characters,
- test with other rules enabled.

Reply:
Sorry for the long delay, somehow this patch got lost and only rediscovered it yesterday. I added the tests.

Still, I am unsure if the coverage failure is a bug in coverage because these new tests would cover that piece of code. Also there are no instructions about running coverage locally for the project so I hope you may be able to provide few hints here.